sopel/modules/reddit.py

```diff
@@ -236,9 +236,58 @@ def comment_info(bot, trigger, match):
     bot.say(message)
 
 
+def subreddit_sorting(bot, trigger, s, sorting):
+    if sorting == 'new':
+        submissions = list(s.new())
+    elif sorting == 'top':
+        submissions = list(s.top())
```

Comment:
```suggestion
        submissions = list(s.top(limit=10))
```

Also worth considering how this feature might allow fetching top posts over a time period other than "all" (the default).